### PR TITLE
New Accounting cheat sheet

### PR DESCRIPTION
**Pull request by mohan08p**
IA page : https://duck.co/ia/view/accounting_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/accounting.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
+{
+    "id": "accounting_cheat_sheet",
+    "name": "Accounting",
+    "template_type": "terminal",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/accounting.json]: This should be using the `reference` template

File: share/goodie/cheat_sheets/json/accounting.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        "Balance Sheet",
+        "Income Statement",
+        "Cash Flow Statement",
+        "Stockholders Equity",
+        "Financial Ratios"
+    ],
+    "sections": {
+        "Financial Statements": [
+            {
+                "key": "balance sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/accounting.json]: nit: We should be title-casing all these `key`s

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=accounting cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/accounting_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/accounting.json [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+                "key": "gross profit",
+                "val": "Sales minus cost of goods sold"
+            },
+            {
+                "key": "cost of goods sold",
+                "val": "Beginning finished goods inventory + net purchases (or cost of goods manufactured) – ending finished goods inventory"
+            },
+            {
+                "key": "selling, general and administrative",
+                "val": "SG&A; operating expenses; noninventoriable costs"
```
Comment (jdorweiler) [inline share/goodie/cheat_sheets/json/accounting.json]: non-inventoriable

**Comment by jdorweiler** [comment]
@mohan08p just a few things to fix and it's ready to go.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=accounting cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/accounting_cheat_sheet_mobile.png)

**Comment by jdorweiler** [comment]
:+1:  thanks @mohan08p

**Comment by daxtheduck** [comment]
@mohan08p this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**